### ISSUE TITLE
Train_csv.py issues

### ISSUE THREAD

**Issue by hoanghoa213** [fizyr/keras-retinanet#258]
`usage: Train.py [-h] [--val_path VAL_PATH] [--weights WEIGHTS]
                [--batch-size BATCH_SIZE] [--gpu GPU]
                train_path classes
Train.py: error: too few arguments`

**Comment by hgaiser** [comment]
I can't really help you without more information. What is the command you are running?

**Comment by hoanghoa213** [comment]
`def parse_args():
    parser = argparse.ArgumentParser(description='Simple training script for object detection from a CSV file.')
    parser.add_argument('train_path', help='Path to CSV file for training (required)')
    parser.add_argument('classes', help='Path to a CSV file containing class label mapping (required)')
    parser.add_argument('--val_path', help='Path to CSV file for validation (optional')
    parser.add_argument('--weights', help='Weights to use for initialization (defaults to ImageNet).',
                        default='imagenet')
    parser.add_argument('--batch-size', help='Size of the batches.', default=1, type=int)
    parser.add_argument('--gpu', help='Id of the GPU to use (as reported by nvidia-smi).')

    return parser.parse_args()`

**Comment by hoanghoa213** [comment]
Can you suggest me to insert into argument?

**Comment by de-vri-es** [comment]
I'm sorry, but it seems that you're not running the latest master of `keras-retinanet`. We don't have a script called `Train.py` (with a capital), nor a script called `train_csv.py`. Please update to the latest master version and try again.
